### ISSUE TITLE
[Feat] Anti log spam

### ISSUE THREAD

**Issue by neoteknic** [marvinlabs/laravel-discord-logger#11]
Add throttle option to log (use Cache / redis cache) to don't spam many time the same error
with 2 options : enabled, throttle time (default 3600 sec)
key is : md5(the error name, file name, and line number)

**Comment by vpratfr** [comment]
Hi

Feel free to submit a PR to support this.

Regards,

**Comment by tarikhagustia** [comment]
i think we don't need to cache the error notification, because we need to fix the error as soon as possible

**Comment by vpratfr** [comment]
I agree it would be a nice thing to have. That error could be muted for an hour and then popup again to avoid too much noise.

**Comment by MrJmpl3** [comment]
I think the AntiLog Spam its great but need be a another package.

**Comment by illnesse** [comment]
Was just looking if someone already suggested log throttling 👍

**Comment by vpratfr** [comment]
> I think the AntiLog Spam its great but need be a another package.

true

**Comment by Chrisx84** [comment]
would be a nice feature since one error in a cronjob can trigger the spam but also good to know that its a critical bug that needs attention immediately.
